inscriptions/ISic001218.xml

```diff
@@ -191,38 +196,20 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="PHI" n="PHI140702">
-                <ab>
-                    <lb n="1"/>
-                    <expan>
-                        <abbr>Ἰαπύρου</abbr>
-                        <ex>?</ex>
-                    </expan>
-                     {²⁷
-                    <supplied reason="lost">Δ</supplied>
-                    <expan>
-                        <abbr>ιαπύρου</abbr>
-                        <ex>?</ex>
-                    </expan>
-                    ;
-                    <supplied reason="omitted">Ζω</supplied>
-                    <expan>
-                        <abbr>πύρου</abbr>
-                        <ex>?</ex>
-                    </expan>
-                    }²⁷
-                    <lb n="2"/>
-                </ab>
-            </div>
-            <div type="edition" xml:space="preserve" subtype="PHI" n="PHI334676">
+            <div type="edition" xml:space="preserve" xml:lang="grc" resp="#MH">
                 <ab>
-                    <lb n="1"/>
-                    <supplied reason="lost">Δ</supplied>ιαπύρου.
-                    <lb n="2"/>
+                    <lb n="1"/><orig>ΙΑΠΥΡΟΥ</orig>
```

Comment:
again, for preference, follow the more recent edition of I.Lipara, at:
https://epigraphy.packhum.org/text/334676